screen/Menu/Menu.style.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from "react-native";
+import colors from "../../res/colors";
+
+export default StyleSheet.create({
+    button: {
+        color: colors.black,
+        fontSize: 20,
+        fontWeight: "bold",
+        height: 40,
+        marginLeft: 16,
+        marginTop: 12
+
+    },
+    body: {
+        backgroundColor: colors.white,
+        borderBottomColor: "gray",
+        borderBottomWidth: 1,
+    },
+    arrow: {
+        position: "absolute",
```

Comment:
wrong, no position absolute and no right property.
your 'body' style should use flex styles for position. Try to find by yourself how to display text and icon in one row and on different sides from each other with flex.
Naming for styles: do you think that naming for button style as 'body' is good and correct by sense ? Try to create more understandable name for this style